### PR TITLE
put internal API partner team is using (FSharp) back

### PR DESCRIPTION
**Pull request by heejaechang**
fixed RPS failure - https://devdiv.visualstudio.com/DevDiv/_git/VS/pullrequest/150702

got broken by recent changes. putting back the API back.

### CODE REVIEW

**Comment by heejaechang** [comment]
tagging @jinujoseph

**Comment by sharwell** [comment]
Did you mean to target **master** or one of the earlier branches?

File: src/Features/Core/Portable/Completion/CommonCompletionItem.cs [hunk 1]
```diff
@@ -1,14 +1,32 @@
...
 using Microsoft.CodeAnalysis.Tags;
 using Roslyn.Utilities;
 
 namespace Microsoft.CodeAnalysis.Completion
 {
-    internal static class CommonCompletionItem 
+    internal static class CommonCompletionItem
     {
+        [Obsolete("This is a compatibility shim for FSharp; please do not use it.", error: true)]
+        public static CompletionItem Create(
```
Comment (jasonmalinowski) [inline src/Features/Core/Portable/Completion/CommonCompletionItem.cs]: @dpoeschl Didn't we already have some other workaround for this? Or am I confusing this with the signature help thing?

Comment (dpoeschl) [reply src/Features/Core/Portable/Completion/CommonCompletionItem.cs]: You're probably thinking of [the SigHelp one](https://github.com/dotnet/roslyn/pull/30844).

**Comment by jinujoseph** [comment]
This is to fix a RPS regression in preview3 - master branch

**Comment by agocke** [comment]
@jinujoseph Looks like the same change may be present in preview2

**Comment by jinujoseph** [comment]
Oh @agocke we did take a merge from p2 to master ?

**Comment by agocke** [comment]
@jinujoseph Yup, Roslyn merges have been moving for a bit now.

**Comment by heejaechang** [comment]
@jasonmalinowski moved to preview2

**Comment by heejaechang** [comment]
@jinujoseph can I check this in?

**Pull request merged**